smallslive/templates/home_banners_mobile.html

```diff
@@ -18,35 +20,52 @@
   <div id="banner-live-box" class="box live">
     <p class="title">Live</p>
     <p class="content">Our <span class="accent-color">live stream</span> has expanded our 60 seat clubs into a
-        <span class="accent-color">worldwide community</span> of jazz fans and supporters.
-        Sign up for free to watch our <span class="accent-color">live streams</span> each night.</p>
+      <span class="accent-color">worldwide community</span> of jazz fans and supporters. Sign up for free to watch our <span
+        class="accent-color">live streams</span> each night.</p>
     <div class="banner-learnmore">
       <button>Become a member</button>
     </div>
   </div>
   <div id="banner-mission-box" class="box mission">
     <p class="title">Mission</p>
     <p class="content">The <span class="accent-color">Smallslive foundation</span> is a
-        <span class="accent-color">not-for-profit</span> organized for the purpose of the
-        <span class="accent-color">dissemination of jazz music</span>
-        through our archive, live streams, sponsorship of individual jall projects & tours,
-        education, and live performances at Smalls &Smezzrow. we are supported by our
-        members through <span class="accent-color">tax-deductible donations</span>.</p>
-      <div class="banner-learnmore">
-        <button>Become a member</button>
-      </div>
+      <span class="accent-color">not-for-profit</span> organized for the purpose of the
+      <span class="accent-color">dissemination of jazz music</span>
+      through our archive, live streams, sponsorship of individual jall projects & tours, education, and live performances at Smalls
+      &Smezzrow. we are supported by our members through <span class="accent-color">tax-deductible donations</span>.</p>
+    <div class="banner-learnmore">
+      <button>Become a member</button>
+    </div>
 
   </div>
+  {% else %}
+  <div id="banner-supporting-box" class="box supporting">
+    <p class="title7">WHY BECOME A <span class="bold"> SUPPORTING MEMBER?</span></p>
+    <p class="content text9">The mission of the <span class="accent-color">SmallsLIVE foundation</span>
+      for jazz arts and education is to <span class="accent-color">disseminate jazz music</span>
+      through recordings and live streams. We sponsor individual jazz projects, tours and educational classes, and subsidize operating
+      expenses for Smalls + Mezzrow jazz clubs. By making a
+      <span class="accent-color">tax deductible donation</span> to the SmallsLIVE Foundation you will get access to our extensive
+      <span class="accent-color">audio/ video archive</span> and directly support the musicians and the clubs.</p>
+    <div class="banner-learnmore">
+      <button onclick='becomeMember()'>Become a member</button>
```

Comment:
This should be `<a href="{% url 'become_supporter' %}">Become a member</a>`

Reply:
Done